src/components/config-tabs/external-data-comparison/DenominatorSelect.js

```diff
@@ -0,0 +1,35 @@
+import { SingleSelectFieldFF, ReactFinalForm } from '@dhis2/ui'
+import React from 'react'
+import { useConfigurations } from '../../../utils/index.js'
+import styles from './DataMappingForm.module.css'
+
+const { Field } = ReactFinalForm
+
+export const DenominatorSelect = () => {
+    const configurations = useConfigurations()
+
+    // filter denominators that have data IDs & sort alphabetically
+
+    const denominatorOptions = React.useMemo(() => {
+        const denominatorsWithDataIds = configurations.denominators
+            .filter((denominator) => denominator.dataID != null)
+            .sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
I got an error when initially opening the app because I had some denominators without names (that however should be a fixed issue). I think we can leave this as is, but we might consider generally opening a ticket to make localeCompares throughout the app use optional chaining to prevent a crash when the comparison property is missing for whatever reason